### PR TITLE
test: add testcase for link priority

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: test/units/testsuite-64.sh [hunk 1]
```diff
@@ -237,17 +254,42 @@ EOF
...
+            if ((disk % 2 == i % 2)); then
+                udevadm lock --device="${devices[$disk]}" sfdisk -q --delete "${devices[$disk]}" &
+            else
+                udevadm lock --device="${devices[$disk]}" sfdisk -q -X gpt "${devices[$disk]}" <"$partscript" &
+            fi
+        done
+
+        # Wait for the above sfdisk commands.
+        for disk in {0..9}; do
+            udevadm lock --device="${devices[$disk]}" true
```
Comment (mrc0mmand) [inline test/units/testsuite-64.sh:267]: I'm just curious - why is the `udevadm lock` call needed here when we called `sfdisk` above with `udevadm lock` as well?

**Review by mrc0mmand — APPROVED** [review]
LGTM, thank you!

File: test/units/testsuite-64.sh (hunk 1 above)
Comment (yuwata) [reply test/units/testsuite-64.sh:267]: The comment here lacks important words: Wait for the above sfdisk commands _to be finished_.

**Comment by yuwata** [comment]
The revised version only update comment, and the previous CI runs are all green. Merging.

**Pull request merged**